docs/pipelineruns.md

```diff
@@ -756,6 +756,24 @@ If used with this `Pipeline`,  `build-task` will use the task specific `PodTempl
 `PipelineTaskRunSpec` may also contain `StepOverrides` and `SidecarOverrides`; see
 [Overriding `Task` `Steps` and `Sidecars`](./taskruns.md#overriding-task-steps-and-sidecars) for more information.
 
+The optional annotations and labels can be added under a `Metadata` field as for a specific running context.
+
+An example for rendering needed secrets with Vault:
+
+```yaml
+spec:
+  pipelineRef:
+    name: pipeline-name
+  taskRunSpecs:
+    - pipelineTaskName: task-name
+      metadata: 
+        annotations:
```

Comment:
please add the labels in this example as well so that it's clear to users that its also supported

Reply:
good catch! let me update.